code/game/objects/items/tools/synth_bracer/synth_bracer.dm

```diff
@@ -0,0 +1,444 @@
+	if(CAN_PICKUP(usr, src))
+		if(!istype(over_object, /atom/movable/screen))
+			return ..()
+
+		if(!usr.is_mob_restrained() && !usr.stat)
+			switch(over_object.name)
+				if("r_hand")
+					if(usr.drop_inv_item_on_ground(src))
+						usr.put_in_r_hand(src)
+				if("l_hand")
+					if(usr.drop_inv_item_on_ground(src))
+						usr.put_in_l_hand(src)
+			add_fingerprint(usr)
+
+/obj/item/clothing/gloves/synth/attackby(obj/item/attacker, mob/living/carbon/human/user)
+	if((istype(attacker, /obj/item/clothing/gloves)) && !(attacker.flags_item & ITEM_PREDATOR))
+		if(underglove)
+			to_chat(user, SPAN_WARNING("[src] is already attached to [underglove], remove them first."))
+			return
+		underglove = attacker
+		user.drop_inv_item_to_loc(attacker, src)
+		to_chat(user, SPAN_NOTICE("You attach the [attacker] to the bracer's wrist-strap."))
+		user.update_inv_gloves()
+	if(internal_transmitter.attached_to == attacker)
+		internal_transmitter.attackby(attacker, user)
+		return
+
+	if(istype(attacker, /obj/item/device/simi_chip))
+		var/obj/item/device/simi_chip/new_chip = attacker
+		var/chips_num = 0
+		for(var/obj/item/device/simi_chip/chip in ability_chips)
+			if(chip.chip_action == new_chip.chip_action)
+				to_chat(user, SPAN_WARNING("[src] already has this circuit chip!"))
+				return
+			if(!chip.secret)
+				chips_num++
+		if(chips_num >= ability_chips_max)
+			to_chat(user, SPAN_WARNING("[src] can't hold another circuit chip!"))
+			return
+		if(user.drop_held_item())
+			new_chip.forceMove(src)
+			ability_chips += new_chip
+			to_chat(user, SPAN_NOTICE("You slot [new_chip] into [src]!"))
+			playsound(src, 'sound/machines/terminal_processing.ogg', 35, TRUE)
+			if(user.gloves && (user.gloves == src))
+				update_actions(SIMI_ACTIONS_RELOAD, user)
+			else
+				update_actions(SIMI_ACTIONS_RELOAD)
+		return
+
+	if(HAS_TRAIT(attacker, TRAIT_TOOL_SCREWDRIVER))
+		//Remove ability chip - give option or remove all?
+		var/turf/T = get_turf(user)
+		if(!T)
+			to_chat(user, "You cannot do that here.")
+			return
+
+		var/removed_chips = FALSE
+		for (var/obj/item/device/simi_chip/chip in ability_chips)
+			if(chip.secret)
+				continue
+			chip.forceMove(T)
+			ability_chips -= chip
+			removed_chips = TRUE
+		if(removed_chips)
+			if(user.gloves && (user.gloves == src))
+				update_actions(SIMI_ACTIONS_RELOAD, user)
+			else
+				update_actions(SIMI_ACTIONS_RELOAD)
+			to_chat(user, SPAN_NOTICE("You pop out the circuit chips from [src]!"))
+		else
+			to_chat(user, SPAN_NOTICE("There are no removable circuit chips in [src]!"))
+		return
+	return ..()
+
+/obj/item/clothing/gloves/synth/attack_hand(mob/living/carbon/human/user)
+	if(istype(user) && user.gloves == src)
+		internal_transmitter.attack_hand(user)
+		return
+	return ..()
+
+/obj/item/clothing/gloves/synth/forceMove(atom/dest)
+	. = ..()
+	if(isturf(dest))
+		internal_transmitter.set_tether_holder(src)
+	else
+		internal_transmitter.set_tether_holder(loc)
```

Comment:
You should be using `/atom/movable/proc/Moved` or the `COMSIG_MOVABLE_MOVED` for this -- not all movement is done by forceMove